bin/run-couchdb-on-travis.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 
 if [ "$SERVER" = "couchdb-master" ]; then
-  # Install CouchDB Master
+  # Install CouchDB 2.X (clustered)
   docker run -d -p 3001:5984 couchdbdev/debian-8-dev --with-haproxy --with-admin-party-please -n 1
```

Comment:
I believe that [couchdbdev/debian-8-dev](https://hub.docker.com/r/couchdbdev/debian-8-dev/) is older than the official [couchdb:latest](https://hub.docker.com/_/couchdb/) image.

Reply:
The dev images will build from the couchdb master source. We may also want to test against couchdb:latest as an independent, stable test against 2.x.